tutoraspects/commands_v1.py

```diff
@@ -165,7 +167,16 @@ def dump_courses_to_clickhouse(options) -> list[tuple[str, str]]:
         "will be attempted to be transformed, but won't be sent to the destination."
     ),
 )
-def transform_tracking_logs(**kwargs) -> list[tuple[str, str]]:
+@click.option(
+    "--deduplicate",
+    is_flag=True,
+    help=(
+        "A flag to determine if the command should run optimize the tables for deduplication."
+        "If present, the OPTIMIZE FINAL query will be applied for all tables that have a "
+        "deduplication strategy."
```

Comment:
Can we add something like: "This should only be added if you believe events will be duplicated such as replaying logs that have already been added. De-duplication can take a very long time to process."